website/screens/components/select/code/SelectCodePage.tsx

```diff
@@ -320,8 +294,8 @@ const sections = [
         content: (
           <>
             <DxcText as="p">
-              You can add icons to the options of the listbox either using a URL
-              or an inline SVG.
+              You can optionally add icons to the options of the listbox either
```

Comment:
I see this paragraph redundant because it is explained in the table and it doesn't really give much new information.

Reply:
Agree!